docs/guide/clustering_aws_s3.md

```diff
@@ -24,6 +24,11 @@ See <http://docs.aws.amazon.com/AmazonS3/latest/dev/s3-access-control.html>.
 The secret key cannot be retrieved again after the key has been created, so don't lose it.
 (However, you can create new keys if needed.)
 
+!!! note
+
+    Make sure that your bucket is configurated as Public to avoid facing 403 errors.
```

Comment:
```suggestion
    Make sure that your bucket is [configured as Public](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/block-public-access-bucket.html) to avoid facing 403 errors.
```